.github/workflows/CI.yml

```diff
@@ -37,37 +52,49 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@latest
-        continue-on-error: ${{ matrix.version == 'nightly' }}
       - run: |
           git config --global user.name Tester
           git config --global user.email te@st.er
       - uses: julia-actions/julia-runtest@latest
-        continue-on-error: ${{ matrix.version == 'nightly' }}
       - uses: julia-actions/julia-processcoverage@v1
-        continue-on-error: ${{ matrix.version == 'nightly' }}
       - uses: codecov/codecov-action@v1
-        continue-on-error: ${{ matrix.version == 'nightly' }}
         with:
           file: lcov.info
-  # docs:
-  # Package need to be registered first
-  #   name: Documentation
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v2
-  #     - uses: julia-actions/setup-julia@v1
-  #       with:
-  #         version: '1'
-  #     - run: |
-  #         git config --global user.name name
-  #         git config --global user.email email
-  #         git config --global github.user username
-  #     - run: |
-  #         julia --project=docs -e '
-  #           using Pkg;
-  #           Pkg.develop(PackageSpec(path=pwd()));
-  #           Pkg.instantiate();
-  #           include("docs/make.jl");'
-  #       env:
-  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #         DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+
+  slack:
+    name: Notify Slack Failure
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.event == 'schedule'
+    steps:
+      - uses: technote-space/workflow-conclusion-action@v2
+      - uses: voxmedia/github-action-slack-notify-build@v1
+        if: env.WORKFLOW_CONCLUSION == 'failure'
+        with:
+          channel: nightly-dev
+          status: FAILED
+          color: danger
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.RSE_SLACK_BOT_TOKEN }}
```

Comment:
I am confusied why this uses the `RSE_SLACK_BOT_TOKEN` and channel `nightly-dev`.
Shouldn't this be the `DEV_SLACK_BOT_TOKEN` or does it just not matter?

Reply:
uh oh, that means some of the other packages are possibly incorrect here as well. I guess I didn't notice that the token AND channel need to be changed for RSE. Good catch

Reply:
this should be DEV anyway

Reply:
Really, cause it seems like more of an RSE problem?

Reply:
More that it was always on nightly dev before.
But it can be nightly-rse.
you're right

Reply:
Lol, I think it was on nightly-dev because it predated nightly-rse :)